src/AIPlayer.java

```diff
@@ -21,6 +27,10 @@ private Throw getRandomSymbolInRange(int range) {
     }
 
     private Optional<Throw> convertToThrow(int throwIdentifier) {
+        if (throwIdentifier <= Throw.values().length){
+            Throw aThrow= Throw.values()[throwIdentifier-1];
+            return of(aThrow);
+        }
         for (Throw aThrow : Throw.values()) {
             if (aThrow.equalsChoice(throwIdentifier)) {
                 return of(aThrow);
```

Comment:
This was mistakenly left in.  I added another commit after this one, which I thought removed this.  Pretty sure I checked too.